spec/services/deals_service_spec.rb

```diff
@@ -27,8 +27,13 @@
 
   describe :create do
     it "returns instance of Deal class" do
-      @deal = build(:deal)
-      expect(client.deals.create(@deal)).to be_instance_of BaseCRM::Deal
+      deal = build(:deal)
```

Comment:
Yes it is not significant.
But I just added a spec below that follows the same convention and I don't want to assign stuff to instance variable cause it does not make sense here (and adds confusion since you start wondering whether this variable is used somewhere else). So I went for local variable and changed this spec so that both of them follow the same pattern.